src/ripple/rpc/handlers/AMMInfo.cpp

```diff
@@ -136,7 +136,7 @@ doAMMInfo(RPC::JsonContext& context)
         {
             Json::Value vote;
             vote[jss::account] = to_string(voteEntry.getAccountID(sfAccount));
-            vote[jss::trading_fee] = voteEntry[sfTradingFee];
+            vote[jss::trading_fee] = voteEntry[~sfTradingFee].value_or(0);
```

Comment:
This can safely be changed back to the original code `vote[jss::trading_fee] = voteEntry[sfTradingFee];`. If the field isn't set, it'll return 0.